app/models/concerns/cable_ready/updatable.rb

```diff
@@ -93,6 +100,15 @@ def enrich_association_with_updates(name, option)
           options: options
         })
       end
+
+      def broadcast_updates(model_class, options)
+        return if skip_updates_classes.any? { |klass| klass >= self }
+        ActionCable.server.broadcast(model_class, options)
+      end
+
+      def skip_updates_classes
+        Thread.current[:skip_updates_classes] ||= []
```

Comment:
Elsewhere in the library, we use `Thread::Local`. Is that something we should look at, here?

https://github.com/socketry/thread-local/tree/master/guides/getting-started

Reply:
My understanding is that this checks for a thread local variable, whereas `Thread::Local` constructs a thread local singleton. IMHO these are two different things, but maybe @hopsoft can chime in?

Reply:
That tracks! I'm satisfied.